src/features/dwn/profile-protocol/profile.schema.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Profile",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "displayName": { "type": "string" }
+  }
```

Comment:
I'm wondering, should we add the `did` to this as well?

I'm not really sure how these records would be stored in relation to @frankhinek's work around managed identities.

* If these records are written against the master DID, there will be multiple Profile records that we'd somehow need to correlate to a ManagedIdentity. We'd need a way to answer the question: "what managedIdentity is this profile for?" A field representing which DID the profile is for could help solve this issue. 

* If each record is written to the DWN against the individual ManagedIdentity DID, we'd need to request the profile from the DWN via the specific DID anyways. A `did` field wouldn't really be useful in this case, as we needed to make a specific request for the Profile for that specific DID anyways.

Reply:
> 

Great call out. I can't quite tell one way or another. I wonder if @frankhinek would know.